src/components/bodhi-dls/cardVoting.js

```diff
@@ -92,6 +92,9 @@ class CardVoting extends Component {
       // Disable the button if request went through
       confirmBtnDisabled = true;
       buttonText = 'Transaction posted';
+    } else if (skipToggle) {
+      confirmBtnDisabled = false;
+      buttonText = 'Finalize';
     } else if (showAmountInput) {
```

Comment:
Hacky code here. Currently only Finalizing phase will set skipToggle to true so buttonText is hard-coded as "Finalize". The reason we need this is that Finalizing phase doesn't have any input or selection so it doesn't need toggle.